app/src/test/java/org/glucosio/android/activity/PreferencesActivityTest.java

```diff
@@ -1,35 +1,29 @@
 package org.glucosio.android.activity;
 
+import org.glucosio.android.DomainConstants;
 import org.glucosio.android.RobolectricTest;
 import org.glucosio.android.db.User;
 import org.junit.Before;
 import org.junit.Test;
-import org.mockito.Mock;
 import org.robolectric.Robolectric;
 
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
-import static org.mockito.MockitoAnnotations.initMocks;
 
 public class PreferencesActivityTest extends RobolectricTest {
     private PreferencesActivity activity;
 
-    @Mock
-    private User mockedUser;
+    private User user = new User(1, "test", "en", "en", 23, "M", 1, DomainConstants.MG_D_L, "", "", "Test", 0, 100);
```

Comment:
I would made User instance final and extract this magic values to more readable constant fields.

Reply:
Made it final